src/test/java/uk/gov/service/notify/ClientIntegrationTestIT.java

```diff
@@ -353,9 +354,10 @@ private SendEmailResponse sendEmailAndAssertResponse(final NotificationClient cl
     }
 
     private SendSmsResponse sendSmsAndAssertResponse(final NotificationClient client) throws NotificationClientException {
-        HashMap<String, String> personalisation = new HashMap<>();
+        HashMap<String, Object> personalisation = new HashMap<>();
         String uniqueName = UUID.randomUUID().toString();
         personalisation.put("name", uniqueName);
+        personalisation.put("list", Collections.singletonList("value"));
```

Comment:
We shouldn't be changing this - this test sends a notification to our test server, which doesn't have this personalisation. It won't fail (notify discards extra personalisation), but it's misleading.